doc/classes/EditorSettings.xml

```diff
@@ -1550,6 +1550,10 @@
 			A comma-separated list of case-sensitive words to highlight in comments. The text will be highlighted in the script editor with the [member text_editor/theme/highlighting/comment_markers/warning_color] color. These must not include spaces or symbols or they will not be highlighted.
 			[b]Note:[/b] This is only implemented in the GDScript syntax highlighter.
 		</member>
+		<member name="text_editor/theme/highlighting/gdscript/bracket_pair_colors" type="PackedColorArray" setter="" getter="">
+			The list of colors to cycle through for colorizing bracket pairs.
+			Leave blank to disable.
```

Comment:
Needs some description of how it works, how it knows what color to use for what bracket pair